ote/src/cljs/ote/app/controller/route/route_wizard.cljs

```diff
@@ -582,12 +585,12 @@
   (and (not (str/blank? name))
        transport-operator-id))
 
-(defn valid-calendar? [route-calendar]
-  (if (or (empty? route-calendar)
-          (and
-            (empty? (get route-calendar :rule-dates))
-            (empty? (get route-calendar ::transit/service-removed-dates))
-            (empty? (get route-calendar ::transit/service-rules)))) false true))
+(defn valid-calendar? [trip-calendar]
+  (if (or
```

Comment:
`(if cond false true)` is the same a `(not cond)`

Reply:
Yes, but I return false from if, if one of those is empty. If I change this to use only "(or", then I would need to add "(not" to every check